research/2022-08-20-introducing-waku-product.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+name:  "Waku Product"
```

Comment:
Do we want to _publicly_ call it Waku Product? As opposed to this being an internal moniker we use to make it clear what we are talking about.

From external people POV, isn't this just "Waku" (as a product?)